app/assets/stylesheets/views/_brexit-landing-page.scss

```diff
@@ -48,40 +56,23 @@ $red: #E61E32;
   @include govuk-responsive-margin(8, "top");
 }
 
-.full-page-width-wrapper--landing-page {
-  border-top: 2px solid govuk-colour("black");
-}
-
 .landing-page__section {
-  margin-bottom: govuk-spacing(8);
   border-top: 2px solid govuk-colour("black");
-}
-
-.landing-page__section-title {
-  padding: govuk-spacing(7) govuk-spacing(2) govuk-spacing(2) 0;
-  margin-bottom: 20px;
+  padding-top: govuk-spacing(7);
 }
 
 .landing-page__guidance_subheader {
   @include govuk-font(19);
-  margin-bottom: 20px;
+  margin-bottom: govuk-spacing(7);
 }
 
 .landing-page__section-list-wrapper {
-  margin: 0 0 govuk-spacing(6) 0;
-  border-top: 1px solid govuk-colour("mid-grey", $legacy: "grey-2");
-  padding-top: govuk-spacing(4);
-}
-
-.landing-page__section-row-title {
-  padding-left: 0;
-}
-
-.landing-page__section-content {
-  padding-left: 0;
-
-  @include govuk-media-query($from: tablet) {
-    padding-right: 0;
+  margin-bottom: govuk-spacing(6);
+  &:before {
```

Comment:
Nitpick: I'd put a newline in before the `before`.

Reply:
Sounds good, added